Classes/BEMCheckBoxGroup.h

```diff
@@ -0,0 +1,39 @@
+//
+//  BEMCheckBoxGroup.h
+//  CheckBox
+//
+//  Created by Cory Imdieke on 10/17/16.
+//  Copyright © 2016 Boris Emorine. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class BEMCheckBox;
+
+@interface BEMCheckBoxGroup : NSObject
+
+/** An array of check boxes in this group.
+ */
+@property (nonatomic, strong, nonnull, readonly) NSArray<BEMCheckBox *> *checkBoxes;
```

Comment:
Didn't realize we were using the order. 
Using an ordered set would make sense then.